css/pages/Input/input.css

```diff
@@ -0,0 +1,90 @@
+.input-container {
+  border-top: 1px solid var(--lightBlue);
+}
+.textarea-container {
+  display: flex;
+  flex-direction: column;
+  padding: 1rem;
+  width: 35rem;
+  height: 25rem;
+}
+.textarea {
+  border: thin solid var(--lightBlue);
+  border-radius: 0.5rem;
+  font-size: 1.3rem;
+  padding: 1.2rem;
+  resize: none;
+  width: 95%;
+  box-shadow: 0.1rem 0.1rem 1rem 0.1rem var(--lightBlue);
+  font-family: inherit;
+  line-height: 1.5em;
+  height: 15rem;
+}
+.textarea:focus {
+  outline: none;
+}
+.textarea-btn {
+  align-self: flex-end;
+}
+.form {
+  background: white;
+  border-radius: 1rem;
+  box-shadow: 0.1rem 0.1rem 1rem 0.1rem #94a3b8;
+  width: 40rem;
+  height: 23rem;
+  padding: 1rem;
+}
+.input {
+  background: #ecfeff;
+  border: thin solid #cbd5e1;
+  border-radius: 1rem;
+  font-size: 1.5rem;
+  padding: 1rem;
+  width: 90%;
+  margin: 1rem 0.5rem;
+}
+.input:focus {
+  outline: none;
+}
+.input-error {
+  border: 1px solid var(--red);
+}
+.msg-error {
+  color: red;
+  display: inline-block;
+  padding: 1rem;
+  font-size: 1.3rem;
+}
+.btn {
```

Comment:
Are you defining button style once again ?

Reply:
It was affecting the homepage btn as well

Reply:
You should then make your button a more generic one rather defining it for every use case .